server/controllers/auth.controller.ts

```diff
@@ -0,0 +1,44 @@
+import jwt from 'jsonwebtoken'
+import bcrypt from 'bcryptjs'
+import User from '../models/User'
+import { SECRET_KEY, JWT_TOKEN_LIFE_TIME } from '../config/constants'
+
+class AuthController {
+    static _generateAccessToken(id) {
+        const payload = {
+            id,
+        }
+
+        return jwt.sign(payload, SECRET_KEY, {
+            expiresIn: JWT_TOKEN_LIFE_TIME,
+        })
+    }
+
+    async post(req, res) {
+        try {
+            const { email, password } = req.body
+            const user = await User.findOne({ email })
+
+            if (!user) {
+                return res.formatResponse(
+                    req.body,
```

Comment:
it's better  send `user` instead of `req.body`

Reply:
In this condition, user is null. I wanted to handle which one the user wanted to authorize.